### DISCUSSION TITLE
Disable pre value age?

### DISCUSSION THREAD

**Discussion by 1Euro7Cent** [jomjol/AI-on-the-edge-device#2848] [Q&A]
Sometimes the esp just gives up and just stops working. Then I unplug the esp and plug back in. from stopping to work until I notice can be multiple days. After reboot the esp just read all ones and put a more than 10000m³ of water used in the system. This happens almost every few months. 
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/67194495/bc82de12-d5f1-4395-8c39-a1c49bd574ac)

The rate too high stuff should prevent stuff like that. But the pre value just gets too old, and it accepts it.

To human sense a raise of 10000m³ should not be reasonable in a few days

I already tried to increase the pre value age. 
For that I looked in the source code and looked at the datatype and googled the max value for it. Then I went and took the next number that was ~50% of the size and rounded it to a nice number.

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/67194495/fad2cf44-9e98-4941-a60d-ad850e810cb3)

but it seems to do nothing

**Comment by caco3** [comment]
Did you try by simply disable that parameter (check box on the left side)?

**Reply by 1Euro7Cent** [reply]
Why didn't I think of that I'll try it
